py/desispec/redshiftqa/desi_per_fiber_qa_stats.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python
```

Comment:
Same comments as for desi_per_fiber_qa_html.py

Reply:
Same` if __name__`  restructuring.